ecs-init/docker/docker.go

```diff
@@ -427,6 +442,48 @@ func getDockerPluginDirBinds() []string {
 	return pluginBinds
 }
 
+// take pathPredicate as an argument for unit testing
+func getCapabilityExecBinds(pathPredicate func(path string, predicate func(fileInfo os.FileInfo) bool) (bool, error)) []string {
+	isDir := func(fileInfo os.FileInfo) bool {
+		return fileInfo.IsDir()
+	}
+	isFile := func(fileInfo os.FileInfo) bool {
+		return !fileInfo.IsDir()
+	}
+	hostResourcesDir := filepath.Join(hostCapabilitiesResourcesRootDir, capabilityExecName)
+	containerResourcesDir := filepath.Join(containerCapabilitiesResourcesRootDir, capabilityExecName)
+
+	binds := []string{}
```

Comment:
thanks for pointing it out! I thought this would not work:

```go
var a []string = nil
b := []string{}
b = append(b, a...)
```

tested and it worked, will change

Reply:
you don't need the explicit `var a []string = nil`... just `var a []string` will do since the default value is `nil`